frontend/src/routes.js

```diff
@@ -11,18 +11,17 @@ import {
   silentRefreshCallback
 } from "./api/auth";
 
-import BaseDashboard from "./components/BaseDashboard";
-
```

Comment:
As @moonflare mentioned above I would suggest also commenting this out instead of deleting it.